### PR TITLE
Remove Donation Page (using external link instead)

### LINKED ISSUE #173: Remove Donate page, link to external url
Remove the donate page for now and link directly to the open collective donation page for Data Umbrella

https://opencollective.com/data-umbrella/contribute/data-science-event-board-37473/checkout.

image

### PR DESCRIPTION
**Pull request by victoriaklo**
## Ticket Description
Fixes #173 

## Description of Changes
Remove the donate page for now and link directly to the open collective donation page for Data Umbrella

https://opencollective.com/data-umbrella/contribute/data-science-event-board-37473/checkout.
image

## Before and After for UI Updates
<-- You should have 2 images minimum, for desktop, 4 for mobile and desktop combined -->

Before:
Screen Shot 2022-09-13 at 4 09 11 PM

After:
Screen Shot 2022-09-13 at 4 07 26 PM

## For PR Reviewer
- [ ] Does this file change the yarn.lock, package.json or package-lock.json file? If so, why?
- [ ] If this pr contains mobile and desktop changes, did you test on IphoneXr and desktop views?
- [ ] Does this file match the related tickets linked figma file, or does it pass the visual smell test?
- [ ] If this file contains javascript, does the javascript pass the smell test?
- [ ] If you don't feel super confident in your review, did you assign someone more senior to double check?

### CODE REVIEW

File: src/components/elements/MobileNav.jsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ function MobileNav() {
             <Link to="/events/new">Post Event</Link><br/>
             <Link to="/events/weekly-digest">Weekly Digest</Link><br/>
             <Link to="/sponsors">Sponsors</Link><br/>
-            <Link to="/donate">Donate</Link><br/>
+            <a href="https://opencollective.com/data-umbrella/contribute/data-science-event-board-37473/checkout">Donate</a><br/>
```
Comment (jtorreggiani) [inline src/components/elements/MobileNav.jsx:67]: Not blocking, but we should pull this URL into a constant that we can import when needed to remove duplication.

File: src/components/elements/Footer.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ function Footer() {
         <div>
           <ul className="grid lg:grid-rows-3 lg:gap-4">
             <li className="mb-2">
-              <a href="https://opencollective.com/data-umbrella">Donate</a>
+              <a href="https://opencollective.com/data-umbrella/contribute/data-science-event-board-37473/checkout">Donate</a>
```
Comment (jtorreggiani) [inline src/components/elements/Footer.jsx:24]: Both links should be "target=_blank" to open the donate link in a new tab.

**Review by jtorreggiani — CHANGES_REQUESTED** [review]
This update is looking good! Added two comments. 1 blocking and 1 style feedback.

**Review by jackwkinsey — APPROVED** [review]

**Comment by jackwkinsey** [comment]
@jtorreggiani Looks like you'll need to sign off on this one as well since you had a review where you requested changes 👍

**Pull request merged**